### ISSUE TITLE
When and how to encode text in manuscript or work records?

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#1570]
This is an issue that has come up repeatedly in the last time, and might benefit from some clarification, also in the guidelines.
The first question is: When should text be transcribed in the manuscript's record, and when in the work's record? A critical edition belongs to the work record, what about a manuscript transcription that might become a critical edition in the future? I have included transcriptions in the corresponding manuscript records, but I see the point of the other perspective as well. Maybe both are possible?
The second issue is: Often, we include text (for incipits, explicits and similar short excerpts) from one manuscript in the work record. This is particularly useful when the manuscript description in question is in another branch that will not be merged for some time. A general practice has been in such cases to mention the manuscript source and include the text without any markup that pertains to its rendering particular to the manuscript (page breaks, rubrication, etc.).

What are your thoughts on these? Could our general practices be mentioned in the guidelines in one or two sentences, or should this left as open as possible for everyone to find individual solutions?

@DenisNosnitsin1970 @DariaElagina @MarcinKrawczuk @eu-genia @PietroLiuzzo @smaugustine

**Comment by DenisNosnitsin1970** [comment]
I think that the text should be transcribed both in the work and in the manuscript records. They are not just text passages, but are instrumentilized and provide arguments supporting text identification.  Yes, I would agree that we include text passages into work records without further mark up, which is for individual witnesses.

**Comment by thea-m** [comment]
Just to clarify: my question about whether to include text in work or manuscript records was mainly related to complete manuscript transcriptions, not smaller parts such as incipits.

**Comment by DenisNosnitsin1970** [comment]
But how can you include the entire transcription? 
Is it technically possible?

**Comment by thea-m** [comment]
In a manuscript, you mean? Yes, you can have a look for example at EMIP01859 or BLorient718

**Comment by DenisNosnitsin1970** [comment]
Well, of course in some cases it will be possible and the transcriptions are wellcome, but it is not the goal, as far as I understand.

**Comment by thea-m** [comment]
No, of course not, the question is just which is the better place for these transcriptions if one happens to have them already

**Comment by DenisNosnitsin1970** [comment]
I see. Do you mean smth like (in case the question is about a full text): if it is a critical edition with a critical text (or, broadening, any scholarly edition), it can be under works (or an Ethiopian "church edition" with undefined witnesses), but if it is a transcription of one specific witness it should be better  placed under that witness.

**Comment by DariaElagina** [comment]
Thank you, @thea-m  for rising this discussion. I think that it might be a good practice, once a transcription of a manuscrpt is already done to paste it to the work record as well. Of course if there is no text. That could increase the searchability. Once an edition will become available, the text can be substituted. And one should always include the source of the text to avoid any misunderstandings.  On the other hand, if one searches for a text not only in the works but in manuscripts as well, it does not matter. Basically, I would leave it open to the encoder.

**Comment by PietroLiuzzo** [comment]
The answer to this, as pointed out already by @thea-m and @DariaElagina, I am afraid is really just a plain and simple "use your judgement", which can be expanded into "use your scientific expertise" and in practical terms as expert encoders it means, stick to the data structure and avoid mixing up concerns if possible.

BM is a research environment for the entire community, we have our project priorities, but these should determine what the Hamburg team works on, not the guidelines we discuss and set up with the community of interested people who also carry out their own projects in BM.

So, I think the question calls for some summary guidance in the GL, if we manage to keep away from providing rules which make people feel they have to do so only and allow them to forget that every tag is a very precise scientific statement they are making. What follows are my tuppence, falling straight into the TL;DR category of posts.

There are some straight forward cases in my opinion that may be documented and are in part already so in the GL.

- the transcription of a manuscript or parts of it, what epigraphists would call a diplomatic edition, which represents the text as is in that particular witness, with all its features, belongs with the manuscript. We can have this because somebody has done it by hand or with transkribus. see https://betamasaheft.eu/Guidelines/?q=transkribus&id=transcription-transkribus This will distinctively be structured by folia and `<pb>`, `<cb>` and `<lb>`. We can have more than one transcription, that of John of Phillip of transkribus, etc. Those added with Transkribus before beeing checked they are generally about 95% correct 
on the follow up question by @DenisNosnitsin1970 on the possibility of complete transcription, we not only can, but we are doing this with transkribus and linking each single line to the exact portion of the image from which it was transcribed.

- an edition of a work, i.e. a critical edition, based on one or more manuscripts (transcriptions) belongs to the work record. This is structured as the text requires. in chapters, verses, etc. it may contain apparatus criticus, etc. is an entirely different beast from the above. Notice that also for the inscriptions we split this two aspects and have edition and transcription, although these may be in most cases for inscriptions simply distinct visualizations of one encoded text. We can have these because some generous people have contributed the fruit of their hard work to the community, before or after its publication in other formats. Here it is however important to stress that the XML edition is a different product from a printed edition. i.e. there is much more to it... we can have more than one edition of a text of course, Kropp 1994, Marrassini 1993, etc. we have put here for example the edited texts from traces. https://betamasaheft.eu/Guidelines/?id=edition describes a deep set of tools to aknowledge different types of critical edition or text provided, the "church edition" would also find its place here and be linked as such to no manuscript, see the witnesses.

of course, if I am starting my encoding of an edition from a main manuscript and have not done intermediate work, I can copy paste from the manuscript to the work record and start editing there! but these two entities, the marked up transcription (with features such as change of hand, deletions, features of the mise en page, etc.) and the edition (with apparatus detailing the sort of divergences between witnesses and encoding of the text which summarizes its features) should be kept usefully distinguished. While the first allows for collation for example and referencing from catalogues, the second is the one I would use to refer to the text for an argument for which I cannot use a manuscript, especially where traditions are differing.

- if we have only transcriptions from others which cannot be linked to images or cannot be checked as transcriptions, like a piece of text copied in the catalogue, then I would go by the source, i.e. if the piece of text is in the catalogue I would still have it in the manuscript `<msContents>`, as part of that work. If a marked up transcription exists these are unnecesarily copied, because the text can be retrieved from <locus> and the new transcription is based on a controllable source, not on the authority of the cataloguer. nothing prevents having both with proper attribution! The catalogue transcription, may be wrong, may have been corrected by you and has its historical relevance also where we are able to provide a newer or alternative transcription. https://betamasaheft.eu/Guidelines/?id=manuscriptContents 

more uncertain cases, in my case are the following

- in same cases we have only a text which cannot be assigned to either, I have in mind the famous list of incipits of the miracles of mary. we copied those into the work records for the reasons outlined by @thea-m and they would have been better in the manuscripts if these where anyhow on the scope of our work, which they are not, as rightly pointed by @DenisNosnitsin1970 in that case we had a list of abstract works to create, and identify so it made more sense to store those few lines in the work file, without encoding partial text structures.

- for short excerpts, as in the example made by Dorothea, I think it is fine to follow a principle of convenience and the above mentioned judgement of the researcher/encoder.

The last two perhaps would do with a few more words in the GL. And in general trying to benefit as much as possible from the transkribus part of workflow would be a good idea IMHO. It is not in our priorities to do so, but it may help if there are large excerpts to transcribe the images automatically and fix as much as appropriate. the workflow is not that long and the model we currently have is acceptable. 

I am strongly against copying over an entire transcription from a manuscript into a work record and thus duplicating it, for several reasons. The socalled edition in the work file would not  be what it claims to be to any extent. If it is a transcription and we have it already, copying it over is not useful and creates ambiguities and confusion, besides being a waste of time, exactly because we have it already. The work and the manuscript will be linked via a msItem/title, and that is sufficient. I think we have to try at least and do not replicate a tradition of mixing up different things which was only related to the printed medium, not to scientific reasons. Also, I see some conceptual difficulties as in most cases only parts of that transcription will be copied to different work records. if you do so for practical or convenience reasons which escape me, certainly the indications suggested by @DariaElagina are vital, i.e. say where the text comes from. https://betamasaheft.eu/Guidelines/?id=work-teiHeader documents several ways to do so according to need. bibl can be linked to divs, notes can be added, relations. just try not to leave it to imagination... 

on the other side I wonder why finding "MS1 which contains W1" and matches a given search query would be better or worst than finding "W1 which is present in MS1" and matches a given search query or indeed of finding "MS1 and W1" which match a given search term and are in a given relation among them if any? it sounds to me entirely related to search expectations, that is to say what you intend to find, which cannot be coded or stored in cookies...

The application does a lot of work to allow you to reach manuscripts from works and viceversa, provided the basic encoding is available, so the un meaningful duplication of text should be unnecessary. if more can be done in the application in this respect, let me know.

**Comment by thea-m** [comment]
Thank you all for the constructive comments. For me, there is no more need for discussion right now, I will make a guidelines PR to you trying to include these thoughts there. Of course, if there are still open questions, please comment here or in the PR.
@PietroLiuzzo, regarding one point raised in your last paragraphs: It is my impression that we are always balancing the wishes for effective encoding with trust in the retrieving possibilities of the app on the one hand with, on the other hand, an inner desire for each record to be as self-contained as possible. Maybe everything will be lost and ONLY THIS ONE RECORD will survive the apocalypse? Who knows? This last desire might be the reason for some occasional redundant encoding.
(is the transkribus model aready public?)

**Comment by eu-genia** [comment]
To add to the confusion, just yesterday I had a discussion, for a different reason, with some IT guys who were wondering why text identification cannot be done automatically ;) While it does sound like a nonsense question to raise, in an ideal future (postapocalyptic) world, if any manuscript can be quickly recognized through an OCR protocol and compared to a clavis containing records for all existing texts - with texts naturally - a certain automatization, at least as a series of suggestions through AI, might become in a way possible. But only if every work record has a text to it. Does not have to be marked up.

**Comment by PietroLiuzzo** [comment]
Thanks @thea-m for the PR, looking forward to it. 
re the model, we trained a 50k+ model, which I hope will be made public soon, I have requested that already.
@eu-genia AI can do a lot with a ground truth, and not only AI, also a simple database... 
The point is we are not even close to having enough of that ground truth. We have a lot of metadata but really, a handful of words when we look at texts... and in concrete probably just a chapter or two of real critical edition encoded as such...
